st-embroider/2023/2023-05-23.md

```diff
@@ -0,0 +1,83 @@
+- EF: this is something to make sure it's working. We currently don't have a fault-tolerant hbs parser
+
+### Addon-blueprint PRs https://github.com/embroider-build/addon-blueprint/pulls <@NullVoxPopuli>
+
+- https://github.com/embroider-build/addon-blueprint/pull/120
+  - EF: try without a name 
+- (discussion about having a non-monorepo test app for a v2 addon)
+
+### release preview automation https://github.com/embroider-build/embroider/pull/1440 <@NullVoxPopuli>
+
+- EF: seems fine
+- A: it would be nice if we could find a way to not automatically add everyone as a watcher so we don't get spammed
+- EF: that seems like a reasonable request, but I feel like notifications are unusable anyway
+- A: a good reason to not make the situation worse
+- EF: a nice follow up to this PR would be to actually do a release on merge. This is why we commit the plan
+
+
+### LinkTo with routable engines and embroider flags turned on cause url not recognised errors <@void-mAlex>
+  
+- https://github.com/embroider-build/embroider/pull/1443
+- From what I can tell ember resolver checks requirejs map for existence of lazy-engine/routes and when it doesn’t find that because static addons is turned on it throws an error making any host app with a declared engine unusable if the app has a linkto an internal engine route
+
+- discussed in pre-meeting office hours
+- A: PR is still failling but I will follow up
+
+
+### node 18+ support and building in ci against all LTS - node 16 EOL in 3 months-ish <@void-mAlex>
+
+- EF: the policy is that we're still supporting 3.28 
+- for CI we can't really test everything at all node versions
+- A: we need to make sure it works in the default node version
+- CM: we should add a tagging system to scenario-tester so we can add a "smoke test" to try a subset of tests against all node versions
+- EF: in the mean time we could open a PR to update to node 18 and not merge it
+
+### merge https://github.com/embroider-build/embroider/pull/1444 <@void-mAlex>
+
+- we don't need patch-package because pnpm has it natively
+- merged
+
+### Hey EmberConf! Video - get involved! <@achambers>
+
+- 2 second clip - do it!
+
+### Where are we going to push the rust implementation to? <@mansona>
+
+- Currently in EF personal https://github.com/ef4/swc https://github.com/ef4/wip-glimmer-swc-plugin https://github.com/ef4/wip-glimmer-swc-demo
+- EF: how many repos do we need? probably better to have our fork and an independent repo for our plugin
+- we can then have github actions to update our patch against upstream swc
+- PS: what about upstreaming this? 
+- CM: wasn't there a talk about sending this to TC39?
+- EF: that's not the first step anyway (content tag spec)
+- there are other potential use cases
+- what I wrote in the draft RFC is quite close what would be a generic content tag
+
+### Do we have any ideas about ember-cli-update for addon blueprint? <@mansona>
+
+- EF: this should be working
```

Comment:
needs testing -- we should probs try adding to our test suite -- fix any bugs in ECU that we run in to